common/python/crypto_utils/signature.py

```diff
@@ -111,7 +111,8 @@ def __encrypt_workorder_indata(
             e_key = item['encryptedDataEncryptionKey'].encode('UTF-8')
 
             if (not e_key) or (e_key == "null".encode('UTF-8')):
-                enc_data = crypto_utility.encrypt_data(data, session_key, session_iv)
+                enc_data = crypto_utility.encrypt_data(
```

Comment:
Do we need an empty line here?

Reply:
@Ram-srini My guess is the line was split into two lines because it was too long.